crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -889,9 +889,10 @@ type DynamicField {
 	"""
 	name: MoveValue
 	"""
-	The actual data stored in the dynamic field.
-	The returned dynamic field is an object if its return type is MoveObject,
-	in which case it is also accessible off-chain via its address.
+	The returned dynamic field is an object if its return type is `MoveObject`,
+	and its contents will be from the latest version that is at most equal to
+	its parent object's version. Otherwise, it is also accessible off-chain
+	via its address.
```

Comment:
This currently reads as if the value is accessible via its address if its return type is *not* `MoveObject`, but it's the other way around, suggesting:

```suggestion
	The returned dynamic field is an object if its return type is `MoveObject`,
	in which case it is also accessible off-chain via its address. Its contents
	will be from the latest version that is at most equal to its parent object's
	version.
```